apps/gitness/src/pages-v2/repo/labels/hooks/use-full-fill-label-store.ts

```diff
@@ -57,9 +59,16 @@ export const useFullFillLabelStore = ({ queryPage, query, enabled = true }: UseF
     setValues(values)
   }, [values, setValues])
 
-  return {
-    isLoading,
-    space_ref,
-    repo_ref
-  }
+  /**
+   * Set loading state to store
+   */
+  useEffect(() => {
+    if (enabled) {
+      return setIsLoading(isDataLoading)
+    }
+
+    setIsLoading(false)
```

Comment:
Best to only ever return teardown functions from a `useEffect`. You can simplify this by not returning early
```suggestion
    setIsLoading(enabled && isDataLoading)
```